cmd/typesctl/providers/ec2_types.go

```diff
@@ -40,25 +40,34 @@ func generateTypesEC2() error {
 		return fmt.Errorf("unable to list EC2 regions: %w", err)
 	}
 
-	// This will throw AuthFailure unless all regions are enabled and "Valid in all AWS Regions"
-	// STS endpoint is configured for the account.
+	// This will throw AuthFailure "AWS was not able to validate the provided access credentials" unless all regions
+	// are enabled and "Valid in all AWS Regions" STS endpoint is configured for the account.
+	//
+	// On some accounts, there is a region that fails to generate with this tool, and it is not visible on the
+	// account setting page and cannot be enabled. For this reason, the tool skips such regions.
 	//
 	// For more info:
 	// https://aws.amazon.com/premiumsupport/knowledge-center/iam-validate-access-credentials/
 	// https://docs.aws.amazon.com/general/latest/gr/rande-manage.html
 	// https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html#sts-regions-manage-tokens
 	for _, region := range regions {
+		fmt.Printf("Generating for region %s", region)
 		client, regionErr := clients.GetServiceEC2Client(ctx, region.String())
 		if regionErr != nil {
-			return fmt.Errorf("unable to list EC2 regions: %w", regionErr)
+			return fmt.Errorf("unable to get regional EC2 client: %w", regionErr)
 		}
 		instTypes, regionErr := client.ListInstanceTypesWithPaginator(ctx)
 		if regionErr != nil {
-			return fmt.Errorf("unable to list EC2 instance types: %w", regionErr)
+			// skip
+			fmt.Printf("unable to list EC2 instance types (region STS not enabled?): %s\n", regionErr.Error())
```

Comment:
Because previously when it encountered a "disabled datacenter" configuration (see the big comment in the file for more details), it stopped the import process. And there is currently a new datacenter I cannot enable on my account for some reason (south africa 2) that makes import process impossible. Now, this only prints a warning and continues importing other DCs.